pkgs/tools/admin/aws-nuke/default.nix

```
@@ -1,21 +1,21 @@
-{ lib
-, buildGoModule
-, fetchFromGitHub
-, installShellFiles
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
 }:
-
 buildGoModule rec {
   pname = "aws-nuke";
-  version = "2.25.0";
+  version = "3.26.0";
```

```suggestion
  version = "3.29.1";
```
This is the latest version.